packages/vite/src/node/server/index.ts

```diff
@@ -1035,7 +1035,12 @@ export function resolveServerOptions(
     middlewareMode: raw?.middlewareMode || false,
   }
   let allowDirs = server.fs?.allow
-  const deny = server.fs?.deny || ['.env', '.env.*', '*.{crt,pem}']
+  const deny = server.fs?.deny || [
```

Comment:
do we want to push the .git in any case even with user provided list, and do we need the glob at start and end or is .git by itself enough?

Reply:
agree thats a separate feature, esp if we look at other defaults too